tests/mitre/test_mitre.sh

```diff
@@ -111,7 +113,11 @@ if [ $SELINUX_ENABLED -eq 0 ]; then
 	test_run "linux-def_selinuxsecuritycontext_test.xml" test_mitre linux-def_selinuxsecuritycontext_test.xml "true"
 fi
 
-test_run "linux-def_inetlisteningservers_test.xml" test_mitre linux-def_inetlisteningservers_test.xml "true"
+if  [ $SENDMAIL_RUNNING = yes ]; then
+	test_run "linux-def_inetlisteningservers_test.xml" test_mitre linux-def_inetlisteningservers_test.xml "true"
+else
+	echo "Skipped the inetlisteningservers test as sendmail service is not running." >&2
```

Comment:
I think we usually return 255 when we skip something, but I don't know if it's possible to use 255 here somehow